docs/devel/pull-requests.md

```diff
@@ -58,10 +48,40 @@ Either the [on call](https://github.com/kubernetes/kubernetes/wiki/Kubernetes-on
 There are several requirements for the submit-queue to work:
 * Author must have signed CLA ("cla: yes" label added to PR)
 * No changes can be made since last lgtm label was applied
-* k8s-bot must have reported the GCE E2E build and test steps passed (Travis, Shippable and Jenkins build)
+* k8s-bot must have reported the GCE E2E build and test steps passed (Travis, Jenkins unit/integration, Jenkins e2e)
 
 Additionally, for infrequent or new contributors, we require the on call to apply the "ok-to-merge" label manually.  This is gated by the [whitelist](https://github.com/kubernetes/contrib/blob/master/mungegithub/whitelist.txt).
 
+### Before sending a pull request
+
+The following will save time for both you and your reviewer:
+
+* Enable [pre-commit hooks](development.md#committing-changes-to-your-fork) and verify they pass.
+* Verify `hack/verify-generated-docs.sh` passes.
+* Verify `hack/test-go.sh` passes.
+
+### Visual overview
+
+![PR workflow](pr_workflow.png)
+
+Other notes
```

Comment:
maybe make each of the two paragraphs here linkable sections so that you can link to them directly when closing PRs?